CHANGELOG.md

```
@@ -1,3 +1,91 @@
+# Unreleased - WSL2/WSLg Compatibility
+
+### ✨ New Platform Support
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)